src/components/iconinitials/iconinitials.scss

```diff
@@ -0,0 +1,44 @@
+.mykn-iconinitials {
+  --brand-color-hue: 253deg;
```

Comment:
We need to alter the current primary color, not redefine it. We want to:

 - Respect the current configured color (may be overridden).
 - Alter it to our needs.


Having access to the HSL (hue satursation ligthness)  values allow the for instance create a lighter variant. of the color. But we may need to do som calc.

If dynamic values don't work we need te try blending with a white layer (:before/:after)